analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -49,15 +49,15 @@ packages:
       value: ""
       algorithm: ""
   vcs:
-    type: "Git"
-    url: "<REPLACE_URL>"
-    revision: "<REPLACE_REVISION>"
-    path: "analyzer/src/funTest/assets/projects/synthetic/spdx/package/libs/curl"
+    type: ""
```

Comment:
Commit message: Please reword to not use past tense ("curl library was"), but imperative style.

Reply:
I'd probably start like:

Instead of referring to the cURL package via a local file committed to VCS, use a remote SPDX document to demonstrate the handling of external URLs.

...

Reply:
>In the expexted output file the vcs

"expected", "VCS"